### ISSUE TITLE
Insertion poin while searching

### ISSUE THREAD

**Issue by LucaSpinosi** [dulleh/akhyou#76]
Hi, it can seem a dull request, but, can you add an insertion point to the search bar ?
Sometimes is very annoying losing track of where are you writing, when changing keyword.

**Comment by LucaSpinosi** [comment]
Sorry i've just realised that the insertion point is red, and with the default red theme it become not noticeable.

**Comment by dulleh** [comment]
Yeah, the problem is that for the main theme the primary colour is the same as the accent colour. Not sure what to do tbh.
